### PR TITLE
Change result output location. Closes #3

### LINKED ISSUE #3: Result output location
CSV files should be written into the workding directory, not ../Results which may not exist depending on where the executable is called from.

Instructions may need tweaking / the plotting script moving etc, and more assumptions may need changing.

Ideally this should also not overwrite files taht exist already without a flag / user input, and not just append to files.

### PR DESCRIPTION
**Pull request by MILeach**

### CODE REVIEW

File: src/main.cu [hunk 1]
```diff
@@ -876,14 +876,14 @@ int main(int argc, const char ** argv) {
         }
 
         // Output parameters to file
-        std::ofstream paramsFile("results/" + experiment.title + "/params.csv");
+        std::ofstream paramsFile(experiment.title + "/params.csv");
```
Comment (ptheywood) [inline src/main.cu:879]: Does this actually work? I don't think ofstream will create a directory if it doesn't exist?

Comment (MILeach) [reply src/main.cu:879]: I'll have a test

Comment (MILeach) [reply src/main.cu]: I've just removed all these as I'm pretty sure they are leftover from my old plotting script and now redundant.

**Review by ptheywood — APPROVED** [review]
I thought you could have just replaced / with _, but if the files are not used / duplicate of what is contained in the main csv then no need to produce them.

**Pull request merged**